docs/resources/packetfabric_ix_virtual_circuit_marketplace.md

```diff
@@ -38,6 +38,7 @@ resource "packetfabric_ix_virtual_circuit_marketplace" "ix_marketplace_conn1" {
     vlan            = var.pf_vc_vlan1
   }
   bandwidth {
+    account_uuid      = var.pf_account_uuid
```

Comment:
same

Reply:
`account_uuid` is a required parameter, however, I added the option to set it up with an env variable in `v0.4.2` so we could actually remove it from the example (https://github.com/PacketFabric/terraform-provider-packetfabric/pull/272). This is how we recommend the user to set `PF_ACCOUNT_ID` in the [getting started page](https://registry.terraform.io/providers/PacketFabric/packetfabric/latest/docs/guides/getting_started) now.